src/tests/topp/IDFileConverter_2_output.idXML

```diff
@@ -93,7 +93,7 @@
 			</PeptideHit>
 		</PeptideIdentification>
 	</IdentificationRun>
-	<IdentificationRun date="2009-05-25T12:33:22" search_engine="SEQUEST" search_engine_version="" search_parameters_ref="SP_1" >
+	<IdentificationRun date="2009-05-25T12:33:23" search_engine="SEQUEST" search_engine_version="" search_parameters_ref="SP_1" >
```

Comment:
could we store millisecond accuracy here or add another attribute 'id="run1"' here? I know this would mean an update to the idXML schema but if we make it an optional attribute that would not be very disturbing. Any thoughts?